### ISSUE TITLE
Force queue of VetCenter Facilities does not include all Outstations, mobiles or CAPS.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#13768]
## Describe the defect
When we force queue VetCetner facilities for a bulk push to lighthouse, it is only doing VetCenters and is not doing outstations, mobiles or CAPs (should not be pushing caps now, but once CAPS are in Facility API we neeed to).
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/b4d3c55e-ae99-4e9c-966e-a30035496da8)

All the logic lives here modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php

## To Reproduce
1. visit https://prod.cms.va.gov/admin/config/va-gov-post-api/facility-force-queue
2. Notice that only Vet Cetnters are counted here.  (code was not updated when we split vet centers into 3 types)

## AC / Expected behavior
- [ ] I expect that when I visit the force queue page, there are options for Outstations, VC mobile ~, and CAPs~
- [ ] I expect that the Outstation and mobile options force queue nodes in all states except archived.
- [ ] ~I expect that the CAPs option  only force queuing PUBLISHED CAPs.  (node status = 1) Because they do not exist in the facility API until they are published in the CMS.~ CAPs need to wait for #13777, per @swirtSJW on [Slack](https://dsva.slack.com/archives/C0FQSS30V/p1687893324053059?thread_ts=1687886741.775019&cid=C0FQSS30V).
- [ ] ~I expect that the CAPs option clearly indicates that it is only force queuing PUBLISHED CAPs.~ 

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/e78594cf-c123-4281-941d-074b3d666c7e)

**Comment by swirtSJW** [comment]
This is blocked by https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13769 because without 13769 we would be force pushing incorrect urls for these very facilities.

**Comment by kmariepat-cityfriends** [comment]
Closing because no longer needed due to work on **Some functions got de-namespaced related to va_gov_post_api#13916**

**Comment by swirtSJW** [comment]
I merged the incomplete PR here because it fixed the related bug of not being able to force push anything.
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14102
